packages/shared/components/modals/ProfileActions.svelte

```diff
@@ -34,6 +58,26 @@
             <Text>{profileName}</Text>
         </div>
         <HR />
+        <div class="flex justify-between items-center p-3">
```

Comment:
@jeeanribeiro this should be displayed only for software wallets, hardware (ledger) profile dont use stronghold